src/main/java/com/skytask/repository/ProductRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.skytask.repository;
+
+import com.skytask.model.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
```

Comment:
why do you need a repository class here?

Reply:
I have it there for tests purposes, I plan to eliminate from this module.